### ISSUE TITLE
[BUG] Fresh install crash on creating default output dir

### ISSUE THREAD

**Issue by parktheredcar** [compose-x/ecs_composex#692]
**Describe the bug**
On first run with no arguments, ecs-compose-x crashes with the following error message

```
...AppData\Roaming\Python\Python312\site-packages\ecs_composex\common\settings.py", line 93, in ComposeXSettings
    default_output_dir = f"/tmp/{dt.utcnow().strftime('%s')}"
                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^
ValueError: Invalid format string
```

**To Reproduce**
1. Install with `pip install ecs-composex --user`
2. run `ecs-compose-x`
3. Error due to invalid strftime format string

**Expected behavior**
Should display help message.

**Logs**
If applicable, add logs of errors using from ECS ComposeX

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Python version 3.12

```
pip show ecs-composex
Name: ecs_composex
Version: 0.23.26
```

**Comment by JohnPreston** [comment]
Thanks @parktheredcar for the report.
I haven't tested with Windows, that might be the reason. I don't even have a windows machine anymore :sweat_smile: 
Let me figure what might be going on there.

**Comment by JohnPreston** [comment]
@parktheredcar I published [0.23.27-rc0](https://pypi.org/project/ecs_composex/0.23.27rc0/) for you to try and if this is working well for you, give me a :+1: and I will do the release :)
